var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -105,6 +105,9 @@ class Adios(AutotoolsPackage):
     #   https://github.com/ornladios/ADIOS/pull/204
     patch('zfp051.patch', when='@1.11.0:1.13.1')
 
+    # Fix a bug in configure.ac that causes issues on certain systems
+    patch('automake.patch')
```

Comment:
Can you quickly verify via `spack install adios@1.13.0`, `1.12.0` etc. if the patch is needed and applies without errors for older version? Otherwise let us just add:
```suggestion
    patch('https://github.com/ornladios/ADIOS/pull/207.patch',
          sha256='<please_add_sha256>', when='@1.13.1')
```

Reply:
It applies cleanly from 1.10.0 (but not 1.9.0). However no versions < 1.12.0 actually had their build complete successfully for me, due to unrelated problems, so I limited the patch to `1.12.0:`.